src/main/java/org/hrorm/Validator.java

```diff
@@ -14,7 +12,7 @@
  * A utility for checking whether or not the configuration of a <code>Dao</code>
  * matches the database schema.
  */
-public class Validator {
+public class Validator extends KeylessValidator {
```

Comment:
This isn't actually where one gets an exception if they DaoBuilder tries to assemble a Keyed Dao w/o a Primary Key- the exception is in IndirectDaoBuilder.buildDao().